stylesheets/commons/Brackets.less

```diff
@@ -793,18 +793,21 @@
 	font-size: 85%;
 }
 
+.brkts-popup-body-match-sidewins-overtime,
 .brkts-popup-body-match-sidewins {
 	padding: 0 0 3px 3px;
 	line-height: 11px;
 
-	.skin-lakesideview & {
-		font-weight: 800;
+	& img {
+		[ data-darkreader-scheme="dark" ] &,
+		.theme--dark & {
+			filter: invert( 1 );
+		}
 	}
 }
 
-.brkts-popup-body-match-sidewins-overtime {
-	padding: 0 0 3px 3px;
-	line-height: 11px;
+.brkts-popup-body-match-sidewins .skin-lakesideview {
```

Comment:
shouldn't this be the other way around?
`.skin-lakesideview .brkts-popup-body-match-sidewins {}`

Reply:
```suggestion
.skin-lakesideview .brkts-popup-body-match-sidewins {
```